pkgs/top-level/perl-packages.nix

```
@@ -20559,6 +20572,21 @@ let
     };
   };
 
+  SubStrictDecl = buildPerlModule {
+    pname = "Sub-StrictDecl";
+    version = "0.005";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/Z/ZE/ZEFRAM/Sub-StrictDecl-0.005.tar.gz";
+      sha256 = "sha256-oSfa52RcGpVwzZopcMbcST1SL/BzGKNKOeQJCY9pESU=";
+    };
+    buildInputs = [ LexicalSealRequireHints ];
```

```suggestion
    propagatedBuildInputs = [ LexicalSealRequireHints ];
```

Runtime dependency needs to be propagated

```
$ perl -MSub::StrictDecl
Can't locate Lexical/SealRequireHints.pm in @INC ...
```

---

Yep, thank you. Fixed.